bentoml/_internal/frameworks/fastai.py

```diff
@@ -0,0 +1,308 @@
+    signatures: ModelSignaturesType | None = None,
+    labels: dict[str, str] | None = None,
+    custom_objects: dict[str, t.Any] | None = None,
+    metadata: dict[str, t.Any] | None = None,
+) -> bentoml.Model:
+    """
+    Save a :code:`fastai.learner.Learner` model instance to the BentoML model store.
+
+    Args:
+        name: The name to give to the model in the BentoML store. This must be a valid
+                    :obj:`~bentoml.Tag` name.
+        learner: :obj:`~fastai.learner.Learner` to be saved.
+        pickle_module: The pickle module to use for exporting the model.
+
+                            .. admonition:: About pickling :obj:`~fastai.learner.Learner`
+
+                            If the :func:`save_model` failed while saving a given learner, then
+                            your learner contains :obj:`~fastai.callback.core.Callback` that is not pickable.
+                            This is due to all FastAI callbacks are stateful, which makes some of them not pickable.
+                            To fix this, Use :func:`Learner.remove_cbs` to remove list of training callback that fails.
+
+        signatures: Signatures of predict methods to be used. If not provided, the signatures default to
+                        ``predict``. See :obj:`~bentoml.types.ModelSignature` for more details.
+        labels: A default set of management labels to be associated with the model. An example is ``{"training-set": "data-1"}``.
+        custom_objects: Custom objects to be saved with the model. An example is ``{"my-normalizer": normalizer}``.
+                            Custom objects are currently serialized with cloudpickle, but this implementation is subject to change.
+        metadata: Metadata to be associated with the model. An example is ``{"bias": 4}``.
+                        Metadata is intended for display in a model management UI and therefore must be a
+                        default Python type, such as :obj:`str` or :obj:`int`.
+
+    Returns:
+        :obj:`~bentoml.Tag`: A tag that can be used to access the saved model from the BentoML model store.
+
+    Example:
+
+    .. code-block:: python
+
+        from fastai.metrics import accuracy
+        from fastai.text.data import URLs
+        from fastai.text.data import untar_data
+        from fastai.text.data import TextDataLoaders
+        from fastai.text.models import AWD_LSTM
+        from fastai.text.learner import text_classifier_learner
+
+        dls = TextDataLoaders.from_folder(untar_data(URLs.IMDB), valid="test")
+
+        learner = text_classifier_learner(dls, AWD_LSTM, drop_mult=0.5, metrics=accuracy)
+        learner.fine_tune(4, 1e-2)
+
+        # Test run the model
+        learner.model.eval()
+        learner.predict("I love that movie!")
+
+        # `Save` the model with BentoML
+        tag = bentoml.fastai.save_model("fai_learner", learner)
+    """
+    if isinstance(learner, nn.Module):
```

Comment:
Should we also ensure it's an instance of `Learner`?

Reply:
make sense, will do